applications/StructuralMechanicsApplication/tests/periodic_test/periodic_linear_test_parameters.json

```diff
@@ -0,0 +1,145 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "test_periodic",
+        "parallel_type"   : "OpenMP",
+        "time_step"       : 1.1,
```

Comment:
please move this to the solver settings, same format as in the fluid:
~~~
time_stepping: {
    time_step
}
~~~

Reply:
I see that structural_mechanics_analysis at line 55 is still looking for `time_step` in `problem_data`..

Reply:
so it will not work if I move `time_step`

Reply:
these lines https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py#L52-L56
are only for backwards compatibility 
=> in case solver does not have the info abt the time-step, then it is copied from problem_data